### PR TITLE
Flex > Bootstrap grid

### LINKED ISSUE #36: Try flex formatting
instead of bootstrap grid

### PR DESCRIPTION
**Pull request by JRJurman**
Check it out, it looks nicer and everything.
Fixes #36 

Slider stuff is still broken (look at #38)

### CODE REVIEW

File: src/client/css/styles.css [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
-  margin-top: 2.5%;
-  margin-bottom: 5%;
+  margin: 4% auto;
+  max-width: 1100px;
+}
+
+.flex-frames {
+  display: flex;
+  justify-content: space-between;
+  flex-flow: row wrap;
```
Comment (chtinahow) [inline src/client/css/styles.css]: At what point to the frames start wrapping?

**Comment by chtinahow** [comment]
:+1:

**Comment by JRJurman** [comment]
Last row is not formatting correctly, will look into it

**Comment by chtinahow** [comment]
:100:

**Pull request merged**